Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Generator.cs

```diff
@@ -40,6 +40,18 @@ public void Execute(GeneratorExecutionContext context)
                     return;
                 }
 
+                // Check to see if any of the current syntax trees of any error diagnostics. If so
```

Comment:
Non blocking: "of any" should be "has any".

Reply:
Fixed